Robinhood/Robinhood.py

```diff
@@ -757,3 +757,79 @@ def place_sell_order(
         """
         transaction = Transaction.SELL
         return self.place_order(instrument, quantity, bid_price, transaction)
+    
+    ##############################
+    #GET OPEN ORDER(S)
+    ##############################
+    
+    def get_open_orders(self):
+        """
+        Returns all currently open (cancellable) orders.
+        If not orders are currently open, `None` is returned.
+        TODO: Is there a way to get these from the API endpoint without stepping through order history?
+        """
+
+        open_orders = []
+        orders = self.order_history()
+        for order in orders['results']:
+            if(order['cancel'] is not None): 
+                open_orders.append(order)
+
+        if len(open_orders) > 0:
+            return open_orders
```

Comment:
Is there somewhat of a pattern in this api to return None if a list of something isn't available? If not, then I think it makes more sense for an api to return an empty list if you ask for open orders and there aren't any.